codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2GoAwayFrame.java

```diff
@@ -87,6 +93,17 @@ public Http2GoAwayFrame setExtraStreamIds(int extraStreamIds) {
     }
 
     @Override
+    public Http2GoAwayFrame setLastStreamId(int lastStreamId) {
+        this.lastStreamId = lastStreamId;
+        return this;
+    }
+
+    @Override
+    public Integer lastStreamId() {
+        return lastStreamId;
```

Comment:
@ejona86 
I would prefer to not have the `last stream id`, as I think `extraStreamIds` is more user friendly ... one should not care about the exact stream id. However, when receiving a `GOAWAY` frame, the `Http2FrameCodec` needs to communicate to the `Http2MultiplexCodec` which streams the `GOAWAY` applies to. Any good ideas on how else to do this?